isis/src/control/apps/findfeatures/FeatureAlgorithmFactory.cpp

```diff
@@ -511,8 +511,14 @@ SharedRobustMatcher FeatureAlgorithmFactory::make(const QString &definition) con
  *      createMatcher(extractor))..
  */
 QStringList FeatureAlgorithmFactory::formatSpecifications(QString specification) const {
-  QStringList parts = specification.split("/", QString::SkipEmptyParts);
-
+  QStringList parts;
+  if ( specification.contains(QRegularExpression("@savepath", QRegularExpression::CaseInsensitiveOption)) ) {
+    QRegularExpression sep("/(?=(.*(@savepath)))",QRegularExpression::CaseInsensitiveOption);
+    parts = specification.split(sep, QString::SkipEmptyParts);
```

Comment:
I just want to confirm: if you are splitting on a regex that includes "@savepath", does this not mean that the parts list would now be missing "@savepath" in it's elements...? I might just be misunderstanding the regex.

Reply:
Confirmed with @paarongiroux that this regex does not strip the "@savepath" from the parts list.